pkg/ansible/flags/flag.go

```diff
@@ -76,4 +81,5 @@ func (f *Flags) AddTo(flagSet *pflag.FlagSet, helpTextPrefix ...string) {
 		"",
 		"Path to installed Ansible Collections. If set, collections should be located in {{value}}/ansible_collections/. If unset, collections are assumed to be in ~/.ansible/collections or /usr/share/ansible/collections.",
 	)
+	flagSet.MarkDeprecated("max-workers", "use --max-concurrent-reconciles instead.")
 }
```

Comment:
Not passing in the CI: 

pkg/ansible/flags/flag.go:84:24: Error return value of `flagSet.MarkDeprecated` is not checked (errcheck)
	flagSet.MarkDeprecated("max-workers", "use --max-concurrent-reconciles instead.")

you need to check the if has error; 

```go
if err := flagSet.MarkDeprecated("max-workers", "use --max-concurrent-reconciles instead."); err != nil {
    ..
}
```

Reply:
To solve: 

```
if err := flagSet.MarkDeprecated("max-workers", "use --max-concurrent-reconciles instead."); err !=nil {
     // panic
}
```